third_party/move/move-vm/runtime/src/storage/ty_layout_converter.rs

```diff
@@ -299,6 +300,37 @@ where
             // delayed fields is needed because enums cannot be delayed fields!
             StructLayout::Variants(variants) => {
                 let mut variant_contains_delayed_fields = false;
+                if ANNOTATED {
```

Comment:
this path is to generate previous layout representation of option from new internal layout of it. According to my understanding, it is explicitly used in string_utils.rs to print out an option value.